docs/CODE_OF_CONDUCT.md

```diff
@@ -76,6 +76,29 @@ address, posting via an official social media account, or acting as an appointed
 representative at an online or offline event. Representation of a project may be
 further defined and clarified by project maintainers.
 
+### AI Generated Contributions Policy
+
+Due to the burden put on maintainers, users submitting multiple low quality pull
+requests, or AI generated comments, reviews, issues, or pull requests, where the
+user does not show a good understanding of what they are posting, might be banned
+from the organisation. Some examples of poor etiquette are:
+
+- Opening a PR without a corresponding issue or for issues which are not yet triaged
+and the "triage" label is not removed;
+- Claiming to work on many issues at the same time;
+- Claiming issues or opening pull requests where another person has already
+  claimed it or where there's already a PR fixing the issue;
+- Opening AI generated pull requests without understanding them;
+- Leaving AI generated comments on issues and pull requests.
```

Comment:
I would suggest omitting the semicolons and even the find period. I would say it's uncommon to see lists punctuated so heavily.